### ISSUE TITLE
Part-time fees appearing in fee table when there is no part time option

### ISSUE THREAD

**Issue by lrhm-unikent** [unikent/of-course#660]
Adam just highlighted the example of'international business management MSc whhich displays PT fees on http://www.kent.ac.uk/finance-student/fees/tuition/#!pg when there is not a part-time delivery for this course at http://www.kent.ac.uk/courses/postgraduate/299/management-international-business

I think on the online prospectus we have some logic to only display the deliveries that are available (e.g. hide PT fees if unavailable) can we add similar logic to the fees table so they appear blank or with N/A?

@aw285-unikent KBS have reported this issue and we'll try to look into it next week on our BAU day. Can you prioritise amongst other issues?

**Comment by eg270-unikent** [comment]
Adam has implemented a workaround for this at the moment, where he has set up adding fee bands so that part-time fees are not displayed. However, due to the current logic these display as TBC. 

@lrhm-unikent can you confirm whether all blank fees should display as NA instead of TBC? Not quite sure how we can differentiate between ones that are blank on purpose and those fees which are yet to be released.

**Comment by eg270-unikent** [comment]
Anything that is written into the CSV files that isn't a number and has a POS code associated with it should be disabled in the data table.
However, we've noticed some cases where Excel has appended a hash symbol to non-numeric fields, i.e. #N/A for social work (MA)

**Comment by eg270-unikent** [comment]
We are awaiting confirmation from Finance regarding overseas part-time fees as N/A. I've put in an issue to look at addressing the inconsistencies between the 2 versions of the snippet:

https://github.com/unikent/programmes-plant/issues/824

**Comment by aw285-unikent** [comment]
@eg270-unikent hello, did this one ever get resolved? I think we're meeting with Adam later in the week so could discuss this then?

**Comment by eg270-unikent** [comment]
Hi @aw285-unikent we implemented the N/A field in the fees table where a full-time/part-time option is unavailable. If there is a delivery but no fees it displays as TBC.

Adam was double checking this with Louise but never got final sign-off. We also created the snippet in the issue I linked to on 20th Feb (see above comment) but never added the snippet code to the 2 data fees tables as we are waiting for the discussion on Thursday before we implement the snippet.
